packages/salesforcedx-vscode-core/src/commands/commands.ts

```diff
@@ -30,74 +30,6 @@ import { taskViewService } from '../statuses';
 import { telemetryService } from '../telemetry';
 import { getRootWorkspacePath, hasRootWorkspace } from '../util';
 
-export class LightningFilePathExistsChecker
```

Comment:
Consolidated the 2 `LightningFilePathExistsCheckers` (one was here and the other was in the `salesforcedx-vscode-lwc` extension) and the FilePathExistsChecker from this file, and moved it to the `baseTemplateCommand.ts` file.